CHANGES.txt

```diff
@@ -145,6 +145,10 @@ Changes
 Fixes
 =====
 
+ - Fix: Restoring empty partitioned table wasn't possible
+
+    - Updated elasticsearch upstream to allow restore of partition templates
```

Comment:
I'd remove it, it's not relevant for a user which parts of the code-base changed in order to fix the issue